fractal_client/cmd/_user.py

```diff
@@ -91,6 +91,15 @@ def user_edit(
 
     user_update = dict()
     if new_email is not None:
+        if (make_verified is False) and (remove_verified is False):
+            # issue 619
```

Comment:
Let's make the comment more informative, e.g. "This check is related to fastapi-users behavior, where `is_verified` is always set to `False` upon an update of `email`."